keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -649,7 +650,7 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
     }
 
     context("artifact creation timestamp exists") {
-      val createdAt = Instant.now()
+      val createdAt = Instant.now().truncatedTo(ChronoUnit.MILLIS)
```

Comment:
Do we still need this with PrecisionSqlSerializer?

Reply:
Short answer yes.  The test needs it because the test is comparing instant.now to the serialized version.  BUT the databases precision since we're limiting the precision doesn't match as a result post operation.  This truncates it to the same level as what the serialization formatter is doing.  I'll add a comment to this effect